airbyte-ci/connectors/pipelines/pipelines/models/reports.py

```diff
@@ -60,6 +60,10 @@ def json_report_remote_storage_key(self) -> str:
     def failed_steps(self) -> List[StepResult]:
         return [step_result for step_result in self.steps_results if step_result.status is StepStatus.FAILURE]
 
+    @property
+    def considered_failed_steps(self) -> List[StepResult]:
+        return [step_result for step_result in self.failed_steps if step_result.consider_in_overall_status]
```

Comment:
Doesn't this just return either full list of all failed steps, or none at all? `consider_in_overall_status` is just a bool that stays the same for all steps, no?

Reply:
It's a `StepResult` property with a default value of `True`, for which we've only overridden it for the `AcceptanceTests` `StepResult`. So it should still see considered failed unit tests or qa checks, for example